pages/2_subscriptions_report.py

```diff
@@ -53,11 +146,124 @@
     with row1_col1:
         st.markdown("**Number of New Subscriptions**")
 
+        # Number of New Subscriptions
+        new_subscriptions_data = subscription_data.groupby('subscription_started_month').size()
+
+        # Convert PeriodIndex to timestamp for better x-axis formatting
+        new_subscriptions_data.index = new_subscriptions_data.index.to_timestamp()
+
+        # Convert Series to DataFrame
+        new_subscriptions_df = new_subscriptions_data.reset_index(name='count')
+        new_subscriptions_df.rename(columns={'index': 'subscription_started_month'}, inplace=True)
+
+        # Create a Plotly line chart
+        fig1 = px.bar(new_subscriptions_df, x='subscription_started_month', y='count')
+
+        # Set bar color
+        fig1.update_traces(marker_color='#306BEA')
+
+        # Update x and y labels
+        fig1.update_layout(
+            xaxis_title='',
+            yaxis_title=''
+        )
```

Comment:
For this chart in particular would you be able to add data labels? Similar to what was done in the example billing report from earlier in the quarter when using this fake data?

![image](https://github.com/user-attachments/assets/3d9bff41-a954-4747-ba89-093720033ea7)

For reference [here](https://github.com/fivetran-streamlit/standardized_billing_model/blob/fa8af19e0c720792af04c06ca119acc522d2b5fd/pages/billing_report.py#L119-L145) is the code used to create this for the revenue by month chart.

Reply:
I don't believe we should add these to the other line charts you have as I believe that will be too noisy with the multiple lines.

Reply:
Updated.